### PR TITLE
PFM-2843--Contract module changes

### PR DESCRIPTION
**Pull request by Vasanth26-egov**

### CODE REVIEW

File: data/state1/works/ContractDocumentType.json
Comment (jagankumar-egov) [inline data/state1/works/ContractDocumentType.json:1]: not required from UI

File: data/state1/works/ContractExecutingAuthority.json
Comment (jagankumar-egov) [inline data/state1/works/ContractExecutingAuthority.json:1]: Not required

File: data/state1/works/ContractorClass.json
Comment (jagankumar-egov) [inline data/state1/works/ContractorClass.json:1]: Not used in UI

File: data/state1/works/ContractDocumentType.json
Comment (Vasanth26-egov) [reply data/state1/works/ContractDocumentType.json:1]: Deleted

File: data/state1/works/ContractExecutingAuthority.json
Comment (Vasanth26-egov) [reply data/state1/works/ContractExecutingAuthority.json:1]: Deleted

File: data/state1/works/ContractorClass.json
Comment (Vasanth26-egov) [reply data/state1/works/ContractorClass.json:1]: Deleted

**Review by jagankumar-egov — APPROVED** [review]

**Review by jagankumar-egov — APPROVED** [review]

**Pull request merged**